en-2016-07-17-latex-template-for-responses-to-referees-reports/statement.tex

```diff
@@ -1,63 +1,64 @@
-%
 % Title:    A LaTeX Template For Responses To a Referees' Reports
 % Author:   Petr Zemek <s3rvac@gmail.com>
 % Homepage: https://blog.petrzemek.net/2016/07/17/latex-template-for-responses-to-referees-reports/
 % License:  CC BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
-%
-\documentclass[a4paper,10pt]{article}
+\documentclass[10pt]{article}
```

Comment:
You generally shouldn't specify the paper size in templates – everyone has different paper settings. Specifying the paper size yourself will give a large group of people sub-optimal printing.